projects/sdk/src/lib/silo/Withdraw.test.ts

```diff
@@ -0,0 +1,80 @@
+import { Source } from "graphql";
+import { sum } from "lodash";
+import { Token } from "src/classes/Token";
+import { TokenValue } from "src/TokenValue";
+import { getTestUtils } from "src/utils/TestUtils/provider";
+import { Withdraw } from "./Withdraw";
+
+const { sdk, account, utils } = getTestUtils();
+
+jest.setTimeout(30000);
+
+describe("Silo Withdrawl", function () {
```

Comment:
After working on Transfer, I think there's value in creating two test cases - sourcing a withdraw from single crate vs multiple crates, to make sure the SDK handles both appropriately.

Reply:
I tried, but this requires forwarding the season twice to get the setup, and this caused the whole test case to run beyond the 30 seconds default.

Reply:
You only need to forward _once_ don't you? Deposit in season n, ffwd to season n+1, deposit again. Your withdrawals could then be sourced from two crates.